lib/workers/repository/update/pr/changelog/source-github.ts

```diff
@@ -14,6 +14,7 @@ import { getTags } from './github';
 import { addReleaseNotes } from './release-notes';
 import { getInRangeReleases } from './releases';
 import { ChangeLogError, ChangeLogRelease, ChangeLogResult } from './types';
+import { slugifyUrl } from './index';
```

Comment:
```suggestion
import { slugifyUrl } from '.';
```
seems we've now circular references? then the function should moved

Reply:
yep, index imports both `getChangeLogJSON`, and the later imports from index.

moved to a new `common.ts` file